### PR TITLE
feat: Visual Code (code server)

### PR DESCRIPTION
**Pull request by victoriavilasb**

### CODE REVIEW

File: .github/workflows/prereleased.yaml [hunk 1]
```diff
@@ -496,6 +604,50 @@ jobs:
...
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ${{ secrets.PRIVATE_CONTAINER_REGISTRY }}
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GHCR_TOKEN }}
+
+    - name: Build and Push VisualCode YData image
+      run: ./images-builder.sh visualcode_ydata gpu ${{ needs.prepare.outputs.version }}
+
```
Comment (portellaa) [inline .github/workflows/prereleased.yaml:650]: 💄  we could move this up, to right after the visual-code-torch, i don't see why rstudio should be in the middle 😅

File: data-science/visualcode/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+ARG TAG
+
+FROM ydata/laboratories-base_python:${TAG}
+
+LABEL ai.ydata.laboratory.visualcode.version=3.9.0
+
+USER root
+
+RUN curl -fsSL https://code-server.dev/install.sh | sh -s -- --dry-run --version=3.9.0 && \
```
Comment (portellaa) [inline data-science/visualcode/Dockerfile:9]: why a dry-run?

File: data-science/visualcode/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    curl -fsSL https://code-server.dev/install.sh | sh
```
Comment (portellaa) [inline data-science/visualcode/Dockerfile:10]: shouldn't you pass the version flat here?

File: data-science/visualcode_torch-1.7/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
+ARG TAG
+
+FROM ydata/visualcode:${TAG}
+
+LABEL ai.ydata.laboratory.visualcode.version=3.9.0
```
Comment (portellaa) [inline data-science/visualcode_torch-1.7/Dockerfile:5]: this is already in the visualcode image

File: data-science/visualcode_tensorflow-2.3/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,13 @@
+ARG TAG
+
+FROM ydata/visualcode:${TAG}
+
+LABEL ai.ydata.laboratory.visualcode.version=3.9.0
```
Comment (portellaa) [inline data-science/visualcode_tensorflow-2.3/Dockerfile:5]: i don't think you need this, this is already done in the visualcode image

File: data-science/visualcode_tensorflow-1.15/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,13 @@
+ARG TAG
+
+FROM ydata/visualcode:${TAG}
+
+LABEL ai.ydata.laboratory.visualcode.version=3.9.0
```
Comment (portellaa) [inline data-science/visualcode_tensorflow-1.15/Dockerfile:5]: same

File: data-science/visualcode/Dockerfile (hunk 2 above)
Comment (victoriavilasb) [reply data-science/visualcode/Dockerfile:9]: it was just to see what was going on in installation process. I will remove it

**Review by portellaa — APPROVED** [review]

**Pull request merged**